monai/networks/utils.py

```diff
@@ -264,8 +265,8 @@ def to_norm_affine(
         raise ValueError(f"affine suggests {sr}D, got src={len(src_size)}D, dst={len(dst_size)}D.")
 
     src_xform = normalize_transform(src_size, affine.device, affine.dtype, align_corners, zero_centered)
-    dst_xform = normalize_transform(dst_size, affine.device, affine.dtype, align_corners, zero_centered)
-    return src_xform @ affine @ torch.inverse(dst_xform)
+    dst_xform = normalize_transform(dst_size, "cpu", affine.dtype, align_corners, zero_centered)
```

Comment:
Is there any case in MONAI that the `affine` is on GPU? It that case, this `GPU -> CPU -> GPU` will be extremely slow and may become a bottleneck.

Thanks.

Reply:
this variable is constructed on cpu, and then moved to the same device as `affine`, so I think the overhead is reasonably small...